### DISCUSSION TITLE
Exception on listDocuments() when orderAttributes contains 'order'

### DISCUSSION THREAD

**Discussion by ddenev** [appwrite/appwrite#3227] [Q&A]
I have a collection that contains an integer attribute named 'order'.
When I call `listDocuments()` like this (Web API):
```js
appwrite.database.listDocuments('budget-categories', [], 100, 0, '', 'after', ['order'], ['ASC'])
```
I get the following exception:
```
Access to XMLHttpRequest at 'http://192.168.88.11/v1/database/collections/budget-categories/documents?limit=100&offset=0&cursor=&cursorDirection=after&orderAttributes%5B0%5D=order&orderTypes%5B0%5D=ASC' from origin 'http://localhost:9000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
AppwriteException: Network request failed
    at Appwrite.<anonymous> (http://localhost:9000/node_modules/.q-cache/vite/spa/deps/appwrite.js?v=af4ab88e:1985:15)
    at Generator.throw (<anonymous>)
    at rejected (http://localhost:9000/node_modules/.q-cache/vite/spa/deps/appwrite.js?v=af4ab88e:495:32)
GET http://192.168.88.11/v1/database/collections/budget-categories/documents?limit=100&offset=0&cursor=&cursorDirection=after&orderAttributes%5B0%5D=order&orderTypes%5B0%5D=ASC net::ERR_FAILED 500
```
This only happens with the 'order' attribute. If I change to any other attribute from the collection, it works (even without indexes defined on the collection).

Is this a bug or am I doing something wrong?

**Comment by ddenev** [comment]
Looks like a bug.

I just created another attribute - 'order1' - same as 'order' (integer, required) and there is no exception. So it seems appwrite is choking on the attribute named 'order'.

**Reply by wrbl606** [reply]
This is also true for attributes named `group` -- with the exact same configuration `group1` works when `group` returns:
`AppwriteException: null, {"message":"Error: Server Error","code":500,"version":"0.14.2"}` in Flutter integration.

**Reply by wrbl606** [reply]
Attributes named `select`, `where`, or `limit` also causes problems so I presume it'll be the case for all MySQL keywords.
